scripts/boot.upstart.conf

```diff
@@ -0,0 +1,7 @@
+description "boot config"
+start on (local-filesystems and net-device-up IFACE!=lo)
+respawn
+env DISPLAY=:0.0
+script 
+	cd /var/goddard/agent && chmod a+x scripts/boot.sh && ./scripts/boot.sh | tee /var/log/goddard.boot.log tty1
+end script
```

Comment:
ping on this @Johanndutoit ? I suspect there'll end up being two upstart-y things now (unless we have the new one overwrite the old one). Is that a problem?